config/plugins/webhooks/news/script.js

```diff
@@ -50,24 +50,27 @@
         el.parentNode.replaceChild(clean, el);
 
         let currentGalaxyVersion = Galaxy.config.version_major;
-
-        // If we're at the 23.1 release candidate, we want to show the 23.0 release notes still.
-        // This should be the last release using this hack -- new notification
-        // system will provide notes moving forward
-
-        if (currentGalaxyVersion == "23.1" && Galaxy.config.version_minor.startsWith("rc")) {
-            currentGalaxyVersion = "23.0";
-        }
-
-        const releaseNotes = `https://docs.galaxyproject.org/en/latest/releases/${currentGalaxyVersion}_announce_user.html`;
         const lastSeenVersion = window.localStorage.getItem("galaxy-news-seen-release");
-        // Check that they've seen the current version's release notes.
-        if (lastSeenVersion != currentGalaxyVersion) {
+
+        // If we're at a deployed release candidate, just mark it seen and show
+        // the previous notes if someone clicks the link.  RC notes won't exist.
+        if (Galaxy.config.version_minor.startsWith("rc")) {
+            // If we, for whatever reason, need to do this again just add
```

Comment:
I guess we should have broken this down into major, minor, micro and suffix, or just return the complete version ... whatever is going to replace the webhook is likely also going to have to deal with this ?

Reply:
So moving forward I expect it'll just be an explicit helper script or a button in the admin notifications UI to 'send release notification' that an admin will be able to get hands on and customize prior to sending.